Packs/CommonScripts/Scripts/SetGridField/SetGridField.py

```diff
@@ -277,8 +277,8 @@ def build_grid_command(grid_id: str, context_path: str, keys: List[str], columns
         raise DemistoException(f'The number of keys: {len(keys)} should match the number of columns: {len(columns)}.')
     # Get old Data
     old_table = get_current_table(grid_id=grid_id)
-    # Normalize columns to match connected words.
-    columns = [normalized_string(phrase) for phrase in columns]
+    # Change columns to all lower case. Not using `normalize()` as underscores are allowed in columns names.
+    columns = [phrase.lower() for phrase in columns]
```

Comment:
what about other special characters that need to be removed (like `$` or `%`)?

Reply:
was just about to commit :)
By the way, the current `normalize` that is applied to keys does not remove them, but on the other hand none of them is an accepted char (unlike `_` for columns). Should I open an issue for it?